res/controllers/Denon-MC7000-scripts.js

```diff
@@ -833,10 +993,37 @@ MC7000.crossfaderAssign = function(channel, control, value, status, group) {
 // Assign Spinback length to STOP TIME knob
 MC7000.stopTime = function(channel, control, value, status, group) {
     var deckNumber = script.deckFromGroup(group);
-    // "factor" for engine.brake()
+    // "factor" for engine.brake() and engine.softStart()
     // this formula produces factors between 31 (min STOP TIME for ca 7 sec back
     // in track) and 1 (max STOP TIME for ca 18.0 sec back in track)
     MC7000.factor[deckNumber] = (1.1 - (value / 127)) * 30 - 2;
+    MC7000.factor2[deckNumber] = (127.69 - value);
+};
+MC7000.lastpress = 0;
+MC7000.play = function(channel, control, value, status, group) {
```

Comment:
this is probably broken because `lastpress` does not take into account that there are multiple play buttons on a controller.

Reply:
actually it was kind of hard to produce a misbehavior in practice, anyway fixed now.